src/modules/FFmpeg/FFDecSW.cpp

```diff
@@ -339,8 +339,6 @@ bool FFDecSW::open(StreamInfo &streamInfo, VideoWriter *)
 		return false;
 	if (codec_ctx->codec_type == AVMEDIA_TYPE_VIDEO)
 	{
-		if (codec->capabilities & CODEC_CAP_DR1)
```

Comment:
IIRC this change will produce incorrect frames (with borders?) for FFmpeg 2.2 which (I hope) still works.